access/jira/config.go

```diff
@@ -67,18 +67,6 @@ func LoadConfig(filepath string) (*Config, error) {
 }
 
 func (c *Config) CheckAndSetDefaults() error {
-	if c.Teleport.AuthServer == "" {
```

Comment:
Can you make the same changes here that I noted in Pagerduty
https://github.com/gravitational/teleport-plugins/pull/220#discussion_r655791392
https://github.com/gravitational/teleport-plugins/pull/220#discussion_r655798486